### PR TITLE
Command: polish "AddTaskParser"

### PR DESCRIPTION
**Pull request by nightism**
Has polished some parts regarding the last PR.

### CODE REVIEW

**Comment by nightism** [comment]
Further polishment is still needed.
Merge is not required if you think it is not needed. @pyokagan

File: src/main/java/seedu/address/logic/parser/AddTaskParser.java [hunk 1]
```diff
@@ -13,10 +19,15 @@
 
     private final AddFloatingTaskParser addFloatingTaskParser;
 
+    private final Command incorrectCommand =
+            new IncorrectCommand(String.format(MESSAGE_INVALID_COMMAND_FORMAT,
+            AddTaskCommand.MESSAGE_USAGE
+            + AddDeadlineCommand.MESSAGE_USAGE
+            + AddEventCommand.MESSAGE_USAGE
+            + AddFloatingTaskCommand.MESSAGE_USAGE));
+
```
Comment (pyokagan) [inline src/main/java/seedu/address/logic/parser/AddTaskParser.java:28]: Ahh okay... I guess this improves the situation somewhat ;-)

**Review by pyokagan — APPROVED** [review]
Thanks.

File: src/main/java/seedu/address/logic/parser/AddTaskParser.java (hunk 1 above)
Comment (pyokagan) [reply src/main/java/seedu/address/logic/parser/AddTaskParser.java:28]: If I had infinite time and energy, I would get rid of the annoying `IncorrectCommand` / `CommandResult` thing and replace them with exceptions.

**Pull request merged**